packages/aa-infra/src/custom/alchemyGasManagerHooks.ts

```diff
@@ -137,139 +77,220 @@ export type AlchemyGasManagerHooks = {
   };
```

Comment:
Need to modify the above as following to allow type inference within params to `createBundlerClient`:

```diff
 estimateFeesPerGas: NonNullable<
-      NonNullable<BundlerClientConfig["userOperation"]>["estimateFeesPerGas"]
+      NonNullable<BundlerClientConfig<Transport, Chain, SmartAccount>["userOperation"]>["estimateFeesPerGas"]
     >;
```